curator-examples/src/main/java/discovery/DiscoveryExample.java

```diff
@@ -192,8 +194,13 @@ private static void listInstances(ServiceDiscovery<InstanceDetails> serviceDisco
                 }
             }
         }
-        finally
```

Comment:
Please keep the closing of resources into the finally block

Reply:
OK, thank you. I'll put the resource shutdown in the finally block.